lib/upfluence/utils/interval_executor.rb

```diff
@@ -0,0 +1,17 @@
+module Upfluence
+  class IntervalExecutor
```

Comment:
nest this class in a `module Utils` no ? to be consistent with the file path

Reply:
arf lol thk